core/src/main/java/com/linecorp/armeria/server/HttpHeaderPathMapping.java

```diff
@@ -146,6 +147,12 @@ public PathMappingResult apply(PathMappingContext mappingCtx) {
         return PathMappingResult.empty();
     }
 
+    // TODO(minwoox) Move this method into MediaType?
+    private static boolean isAnyType(MediaType contentType) {
+        // Ignores the quality factor.
```

Comment:
nit: `Ignores all parameters including the quality factor`? Because there's a method `MediaType.belongsTo()` which ignores the quality factor only.